packages/remix/src/build.ts

```diff
@@ -149,17 +150,37 @@ export const build: BuildV2 = async ({
       join(DEFAULTS_PATH, 'entry.server.jsx'),
       join(appDirectory, 'entry.server.jsx')
     );
-    if (!pkg.dependencies['@vercel/remix-entry-server']) {
+    if (!pkg.dependencies['@vercel/remix']) {
+      // Dependency version resolution logic
+      // 1. Users app is on 1.9.0 -> we install the 1.10.0 (minimum) version of our fork (`@vercel/remix`)
+      // 2. Users app is on 1.11.0 (a version greater than 1.10.0 and less than the latest version of the fork) -> we install the (matching) 1.11.0 version of our fork
+      // 3. Users app is on something greater than our latest version of the fork -> we install the latest version of our fork
+
+      // remixVersion is the version of the `@remix-run/dev` package in the *users' app*
+      const usersRemixVersion = semver.gt(remixVersion, '1.10.0')
+        ? remixVersion
+        : '1.10.0'; // our minimum supported version is 1.10.0
```

Comment:
Can we define this as a const at the top of the file?

Reply:
I meant the `"1.10.0"` specifically

Reply:
gotcha!